.gitignore

```diff
@@ -0,0 +1,68 @@
+#Maven
+
+target/
```

Comment:
Hier gab es ein Missverständnis: Die `target/` Zeile war richtig. Aber die Dateien in dem Ordner, die Teil dieses Pull Requests sind, sind fehl am Platze.

Du hast bisher die (korrekte) Zeile rausgenommen und die (überflüssigen) Dateien dringelassen.